views/default-server-error.html

```diff
@@ -2,7 +2,7 @@
 <html lang="">
   <head>
     <meta charset="utf-8" />
-    <link rel="stylesheet" href="https://monitor.firefox.com/dist/app.min.css">
+    <link rel="stylesheet" href="https://monitor.firefox.com/dist/app.css">
```

Comment:
I don't know for sure – I left it absolute because I assumed the previous dev had a reason.  In past lives I've seen absolute paths or even inline CSS used to make sure there is no obstacle to delivering an error page, but agree with you that this is odd...